File: e2e/dotnet_projects_net10.0/basic_csharp/libtest.cs
using BasicCsharp;
using NUnit.Framework;

[TestFixture]
public sealed class LibTests
{
    [Test]
    public void GetMessage_ReturnsExpectedMessage()
    {
        Assert.AreEqual("Hello from .NET 10.0 library!", Lib.GetMessage());
    }
}

File: e2e/dotnet_projects_net10.0/basic_fsharp/libtest.fs
module BasicFsharp.Tests

open NUnit.Framework
open BasicFsharp

[<TestFixture>]
type LibTests() =
    [<Test>]
    member _.GetMessage_ReturnsExpectedMessage() =
        Assert.AreEqual("Hello from .NET 10.0 F# library!", Lib.getMessage())
